test/TropicalGeometry/tropical_polyhedra.jl

```diff
@@ -0,0 +1,244 @@
+polymake_version = pkgversion(Polymake.polymake_jll)
+
+for minOrMax in (min,max)
+__sign = minOrMax == min ? 1 : -1
+oo = minOrMax == min ? inf : -inf
+
+@testset "TropicalPolyhedron{$minOrMax}" begin
+  TT = tropical_semiring(minOrMax)
+  pts1_a = TT[0 -1__sign 0; 0 -4__sign -1__sign]
+  pts1_b = [
+   TT.(__sign*[0, -1, 0]),
+   TT.(__sign*[0, -4, -1]),
+   TT.(__sign*[0, -3, 0])
+  ]
+  P1_a = tropical_convex_hull(pts1_a)
+
+  @test P1_a isa TropicalPolyhedron{typeof(minOrMax)}
+  @test ambient_dim(P1_a) == 2
+  @test dim(P1_a) == 1
+  @test issetequal(vertices(P1_a), eachrow(pts1_a))
+  @test n_vertices(P1_a) == 2
+  @test issetequal(pseudovertices(P1_a), pts1_b)
+  @test n_pseudovertices(P1_a) == 3
+  
+  @test is_bounded(P1_a) == true
+
+  @test Oscar.pm_object(P1_a) == Oscar.pm_object(P1_a |> tropical_point_configuration)
+
+  cov1 = maximal_covectors(P1_a)
+  @test issetequal(cov1, [IncidenceMatrix([[1], [2], [2]]), IncidenceMatrix([[1], [2], [1]])])
+
+  covdec1_a = covector_decomposition(P1_a)
+  @test issetequal(covdec1_a |> vertices,
+    Vector{QQFieldElem}[
+      __sign*[-1, 0],
+      __sign*[-4, -1],
+      __sign*[-3, 0]
+    ])
+  @test issetequal(covector_decomposition(P1_a; dehomogenize_by=2) |> vertices,
+    Vector{QQFieldElem}[
+      __sign*[1, 1],
+      __sign*[4, 3],
+      __sign*[3, 3]
+    ])
+  if minOrMax == min
+    @test maximal_polyhedra(IncidenceMatrix, covdec1_a) == IncidenceMatrix([[1,3],[2,3]])
+  else
+    @test maximal_polyhedra(IncidenceMatrix, covdec1_a) == IncidenceMatrix([[1,2],[1,3]])
+  end
+
+  ## Adding redundant points gives same tropical polytope with same properties
+  P1_b = tropical_convex_hull(pts1_b)
+  @test issetequal(P1_a |> vertices, P1_b |> vertices)
+  @test dim(P1_b) == 1
+
+  pts2 = TT[0 1__sign 2__sign; oo 0 3__sign; oo oo 0]
+  P2 = tropical_convex_hull(pts2)
+  @test ambient_dim(P2) == 2
+  @test dim(P2) == 2
+  # the following are unbroken with polymake v4.14
```

Comment:
Polymake 4.14 has arrived in Oscar in the meantime, so these comments (and maybe also the polymake version checks in this file) are obsolete

Reply:
> Polymake 4.14 has arrived in Oscar in the meantime, so these comments (and maybe also the polymake version checks in this file) are obsolete

Comments removed; version checks (for test cases) left in place for now.